manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -144,7 +145,7 @@ spec:
           - name: PILOT_ENABLE_PROTOCOL_SNIFFING_FOR_INBOUND
             value: "{{ .Values.pilot.enableProtocolSniffingForInbound }}"
 {{- if .Values.global.istiod.enabled }}
-          - name: WEBHOOK
+          - name: VALIDATION_WEBHOOK_NAME
             value: istio-sidecar-injector
```

Comment:
this looks wrong?  istio-sidecar-injector is not the validation webhook name. We probably wanted to add a new var here?

Reply:
nit: Also can we default it in the code so its not required here?

Reply:
Oops, yes - thanks. Fixed the naming, changed the default to use istiod, and removed the envvar from the manifest.